### DISCUSSION TITLE
https://steinkamp.us/posts/2024-07-27_why-did-the-runner-cross-the-road

### DISCUSSION THREAD

**Discussion by giscus** [zsteinkamp/steinkamp.us#13] [Announcements]
# https://steinkamp.us/posts/2024-07-27_why-did-the-runner-cross-the-road

A little post giving my $0.02 on what side of the road a runner should run on.

https://steinkamp.us/posts/2024-07-27_why-did-the-runner-cross-the-road

**Comment by Dsteinkamp** [comment]
Good job, Zack!  I love the illustrations!
